certbot/main.py

```diff
@@ -16,6 +16,7 @@
 import certbot
 
 from certbot import account
+from certbot import cert_manager
```

Comment:
Well, we don't have a UI anymore, so maybe the point is moot? My idea was that these are the tasks that let the use perform certificate management. Perhaps they could be selected from a list of possible tasks, if we really want a UI. Any good suggestions for alternate naming schemes?